numba/tests/test_array_methods.py

```diff
@@ -179,8 +179,8 @@ def array_imag(a):
     return np.imag(a)
 
 
-def np_unique(a):
-    return np.unique(a)
+def np_unique(a, return_counts=False):
```

Comment:
This won't work as expected as kwargs are not supported in `jit` decorated functions.